@types/game/prototypes/construction-site.ts

```diff
@@ -0,0 +1,30 @@
+declare module "game/prototypes" {
+  import { BuildableStructureConstant } from "game/constants";
+  /**
+   * A site of a structure which is currently under construction.
+   */
+  interface ConstructionSite<T extends BuildableStructureConstant = BuildableStructureConstant> extends RoomObject {
+    readonly prototype: ConstructionSite;
```

Comment:
I would assume a structure type property is present indicating the type of cSite?, Perhaps we should introduce an OwnedRoomObject instead of OwnedStructure, not sure about that

Reply:
Yeah, I would think so as well. I didn't see anything in the docs though, so I just included what they had documented. https://steamcommunity.com/app/1137320/discussions/2/3132793555949745320/